showcase/src/app/app.component.html

```diff
@@ -1,29 +1,68 @@
-<header class="stark-header aah-stark">
+<header class="stark-app-header">
 	<div class="stark-container">
 		<div class="stark-app-bar">
-			<button color="white" mat-icon-button>
-				<mat-icon starkSvgViewBox svgIcon="home"></mat-icon>
-			</button>
-			<button color="white" mat-icon-button (click)="sidenav.toggle()">
-				<mat-icon starkSvgViewBox svgIcon="menu"></mat-icon>
-			</button>
-			<button color="white" mat-icon-button>
-				<mat-icon starkSvgViewBox svgIcon="arrow-left"></mat-icon>
-			</button>
+			<div class="stark-content-left">
+				<div class="stark-actions">
+					<button class="stark-home-button" color="white" mat-icon-button>
+						<mat-icon starkSvgViewBox svgIcon="home"></mat-icon>
+					</button>
+					<button color="white" mat-icon-button (click)="sidenav.toggle()">
+						<mat-icon starkSvgViewBox svgIcon="menu"></mat-icon>
+					</button>
+					<button color="white" mat-icon-button>
+						<mat-icon starkSvgViewBox svgIcon="arrow-left"></mat-icon>
+					</button>
+				</div>
+				<div class="stark-page-title">
+					<h1>STARK</h1>
+				</div>
+			</div>
 
-			<stark-app-logo></stark-app-logo>
+			<div class="stark-content-center">
+				<div class="app-logo">
+					<p>App Logo</p>
+				</div>
+			</div>
 
-			<div class="stark-header-extra">
-				<button color="alert" mat-mini-fab>
-					<mat-icon starkSvgViewBox svgIcon="power"></mat-icon>
-				</button>
+			<div class="stark-content-right">
```

Comment:
Maybe this class could be renamed to `stark-app-bar-content-right` to be more descriptive.